service.rate/lib/client.go

```diff
@@ -11,6 +11,11 @@ import (
 	"google.golang.org/grpc/metadata"
 )
 
+type RatePlanReply struct {
+	RatePlans []*pb.RatePlan
```

Comment:
The most straightforward way to resolve this, is to alias pb.RatePlan in this package, in order to re-export it.

```go
type RatePlan pb.RatePlan

type RatePlanReply struct {
  RatePlans []*RatePlan
  // ...
}
```